bower.json

```diff
@@ -8,6 +8,7 @@
     "chai": "1.8.1",
     "cocktail": "0.5.9",
     "easteregg": "https://github.com/mozilla/fxa-easter-egg.git#71e61f05ab7c0b7c7fcef8ced12d1969503f5968",
+    "fxa-checkbox": "vladikoff/fxa-checkbox#7ec54991ff3e451317b02be8f75a495971232f0a",
```

Comment:
@vladikoff if this is specific to FxA, let's move it under the "mozilla" github org; if not, let's name it something other than "fxa-" :-)